docs/conf.py

```diff
@@ -0,0 +1,161 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import codecs
+import os
+import re
+
+
+def read(*parts):
+    """
+    Build an absolute path from ``parts`` and return contents of the
+    resulting file. Assumes UTF-8 encoding.
+    """
+    here = os.path.abspath(os.path.dirname(__file__))
+    with codecs.open(os.path.join(here, *parts), 'rb', 'utf-8') as f:
+        return f.read()
+
+
+def find_version(*file_paths):
+    """
+    Build a path from ``file_paths`` and search for ``__version__``.
+    """
+    version_file = read(*file_paths)
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                              version_file, re.M)
+
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError('Unable to find version string.')
+
+
+# -- General configuration ------------------------------------------------
+
+# Sphinx extension modules
+extensions = [
+    # include documentation from docstrings
+    'sphinx.ext.autodoc',
+    # highlight todo directives in doc compilation
+    'sphinx.ext.todo',
+    # docstring coverage
+    'sphinx.ext.coverage',
+    # link to view rST code
+    'sphinx.ext.viewcode',
+    # Google-style docstring formatting
+    'sphinx.ext.napoleon',
+    # link to other projects' documentation
+    'sphinx.ext.intersphinx',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'gordon'
+copyright = '2017, Spotify AB'
+author = 'Matt Obarzanek, Lynn Root, Amy Skinner, Steven Stone, James Wen'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The full version, including alpha/beta/rc tags.
+release = find_version('../gordon/__init__.py')
+# The short X.Y version.
+version = release.rsplit(u'.', 1)[0]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = ['_build']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
```

Comment:
This is the default, but I liked it most since it was the cleanest IMO. But if others prefer [a different theme](http://www.sphinx-doc.org/en/stable/theming.html#builtin-themes), I'm fine with that.